### DISCUSSION TITLE
Other deployment options (Github/Azure DevOps)

### DISCUSSION THREAD

**Discussion by mahalel** [google/docsy#690] [Ideas]
Hi, just wondering if there's a chance of having documentation on other ways to deploy, ie. via Github or Azure DevOps.

Love the project by the way, looks so amazing!

**Comment by deining** [comment]
Maybe this is of help for you: recently, I wrote about [Deployment with Amazon S3 + Amazon CloudFront](https://www.docsy.dev/docs/deployment/#deployment-with-amazon-s3--amazon-cloudfront), this is now part of the official dosys documentation (#788).
